### ISSUE TITLE
IDE0060 (Remove unused parameter) false positive for interpolated string handler argument

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/roslyn#76578]
**Version Used**:

VS 17.13 Preview 1

**Steps to Reproduce**:

```csharp
public static void M(bool x, [InterpolatedStringHandlerArgument(nameof(x))] ref MyInterpolatedStringHandler y)
{
    // Do something with message.
}
```

**Diagnostic Id**: IDE0060 (Remove unused parameter)

**Expected Behavior**:

No diagnostic

**Actual Behavior**:

False positive diagnostic
